apisix/plugins/http-logger.lua

```diff
@@ -44,6 +44,7 @@ local schema = {
         inactive_timeout = {type = "integer", minimum = 1, default = 5},
         batch_max_size = {type = "integer", minimum = 1, default = 1000},
         include_req_body = {type = "boolean", default = false},
+        include_resp_body = {type = "boolean", default = false},
```

Comment:
It would be better to use `include_resp_body_expr` to include the response body dynamically. We can eval it in the header_filter.